db/active_replicator_config.go

```diff
@@ -49,6 +49,10 @@ type ActiveReplicatorConfig struct {
 	Continuous bool
 	// PassiveDBURL represents the full Sync Gateway URL, including database path, and basic auth credentials of the target.
 	PassiveDBURL *url.URL
+	// PassiveDBUsername is the name of the user who has access to the remote Sync gateway database.
+	PassiveDBUsername string
+	// PassiveDBPassword is the password for accessing the remote Sync gateway database.
+	PassiveDBPassword string
```

Comment:
These aren't used because we're putting the username+password pair into `PassiveDBURL` before we create the replicator.